source/backend/api/Services/DocumentQueueService.cs

```diff
@@ -156,6 +156,11 @@ public async Task<PimsDocumentQueue> PollForDocument(PimsDocumentQueue documentQ
                 this.Logger.LogError("Document Queue {documentQueueId} is not in valid state, aborting poll.", documentQueue.DocumentQueueId);
                 return databaseDocumentQueue;
             }
+            else if (databaseDocumentQueue.DocumentQueueStatusTypeCode == DocumentQueueStatusTypes.PENDING.ToString() || databaseDocumentQueue.DocumentQueueStatusTypeCode == DocumentQueueStatusTypes.SUCCESS.ToString())
+            {
+                this.Logger.LogError("Document Queue {documentQueueId} is not in valid state, aborting poll.", documentQueue.DocumentQueueId);
+                return databaseDocumentQueue;
+            }
```

Comment:
is this change relevant? I recall a previous PR where these exact lines where replaced with lines 155-158 above...
please clarify